lib/features/auth/presentation/pages/login_form.dart

```diff
@@ -0,0 +1,239 @@
+import 'package:flutter/material.dart';
+import 'package:font_awesome_flutter/font_awesome_flutter.dart';
+import 'package:uome2/core/themes/app_colors.dart';
+
+class LoginForm extends StatefulWidget {
+  const LoginForm({Key? key}) : super(key: key);
+
+  @override
+  State<LoginForm> createState() => _LoginFormState();
+}
+
+class _LoginFormState extends State<LoginForm> {
+  bool _passwordVisible = true;
+
+  @override
+  Widget build(BuildContext context) {
+    return Container(
+      height: MediaQuery.of(context).size.height * 0.95,
+      decoration: const BoxDecoration(
+        color: AppColors.lightGrey,
+        borderRadius: BorderRadius.only(
+          topLeft: Radius.circular(15),
+          topRight: Radius.circular(15),
+        ),
+      ),
+      child: Padding(
+        padding: const EdgeInsets.all(30),
+        child: Column(
+          mainAxisSize: MainAxisSize.min,
+          children: <Widget>[
+            Row(
+              mainAxisAlignment: MainAxisAlignment.end,
+              children: [
+                InkWell(
+                  onTap: () => Navigator.pop(context),
+                  child: const Icon(Icons.cancel_outlined),
+                ),
+              ],
+            ),
+            loginForm(context)
+          ],
+        ),
+      ),
+    );
+  }
+
+  Padding divider() {
+    return Padding(
+      padding: const EdgeInsets.symmetric(vertical: 20),
+      child: Row(
+        mainAxisAlignment: MainAxisAlignment.center,
+        children: [
+          Expanded(
+            child: Container(
+              color: Colors.black.withOpacity(0.3),
+              height: 1,
+            ),
+          ),
+          const Text(' OR '),
+          Expanded(
+            child: Container(
+              color: Colors.black.withOpacity(0.3),
+              height: 1,
+            ),
+          ),
+        ],
+      ),
+    );
+  }
+
+  Padding emailTextField() {
+    return Padding(
+      padding: const EdgeInsets.symmetric(
+        vertical: 10,
+      ),
+      child: TextField(
+        decoration: InputDecoration(
+          contentPadding: const EdgeInsets.only(
+            left: 14,
+            bottom: 8,
+            top: 8,
+          ),
+          enabledBorder: UnderlineInputBorder(
```

Comment:
Check this behavior

Reply:
i just checked and it is better to leave it as it is